pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -110,6 +111,10 @@ func (n *NodeLabeller) Run(threadiness int, stop chan struct{}) {
 
 	n.logger.Infof("node-labeller is running")
 
+	if !n.hasTSCCounter() {
+		n.logger.Error("failed to get tsc cpu frequency, will continue without the tsc frequency label")
+	}
```

Comment:
Since this is an optimization, please consider introducing it in a follow-up commit:
1. It won't be lost in the bigger change.
2. It would have reasoning.

Reply:
I think that this was actually there before the PR have a look in `pkg/virt-handler/node-labeller/node_labeller.go` removed code

Reply:
It was, but I agree with @orelmisan that it would be a good idea to add a comment about it moving out of `prepareLabels` to avoid redundant log messages.